.github/workflows/ci.yml

```diff
@@ -83,11 +97,16 @@ jobs:
       - uses: ./github-actions/setup-bazel-remote-exec
         with:
           bazelrc: .bazelrc
+      - uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # tag=v3.6.0
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
       - uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 # tag=v3.3.1
         with:
           path: |
-            .yarn/cache
-            .yarn/install-state.gz
-          key: v1-${{hashFiles('yarn.lock')}}
+            ~/Library/Caches/bazelisk
+          key: ${{ github.job }}-${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
```

Comment:
Bazlisk cache should be guarded by the bazelversion file I think

Reply:
Do we know that this is where the bazel repository cache lives? IIRC we set an explicit bazelrc option for CircleCI here

Reply:
Whoops, yes it should be the bazelversion file.

Reply:
Added a TODO for the `--repository_cache` flag stuff.  I think its something we will need to put into the github action we use, but want to really consider how we go about it, because this caching might also want to move into it as well.